frontend/src/views/Files.vue

```diff
@@ -69,6 +70,12 @@ export default {
   },
   watch: {
     $route: function (to, from) {
+      if (to.path.endsWith("/") && from.path.startsWith(to.path) && from.path.length > to.path.length ){
+        this.parentDir = "true";
+      } else {
+        this.parentDir = "false"; 
+	window.sessionStorage.removeItem(to.path);
+      } 
```

Comment:
We only need to keep the position when we're back from a subfolder. This is for it.